### PR TITLE
Add per site db user

### PR DESCRIPTION
**Pull request by kirtangajjar**
Issue ref - https://github.com/EasyEngine/site-wp-command/issues/18

closes https://github.com/EasyEngine/site-wp-command/issues/18

Signed-off-by: Kirtan Gajjar <kirtangajjar95@gmail.com>

### CODE REVIEW

File: src/Site_WP_Command.php [hunk 1]
```diff
@@ -242,6 +242,17 @@ public function create( $args, $assoc_args ) {
...
 
+	/**
+	 * Creates database user for a site
+	 *
+	 * @param string $site_url URL of site
+	 *
+	 * @return string Generated db user
+	 */
+	private function create_site_db_user( string $site_url ) : string {
+		return $site_url . '-' . \EE\Utils\random_password( 6 );
```
Comment (rahulsprajapati) [inline src/Site_WP_Command.php]: @kirtangajjar Can you please add some limitation to the site_url string length?
We are currently using mariadb and it has database user name limit to 80 char.
> Users have a maximum length of 80 characters.

ref: https://mariadb.com/kb/en/library/identifier-names/#maximum-length

and here our `site_url` can have more then 80 char - considering the site url might be like `example.com` `test.example.com` `test.example.com/test/` ...

Comment (kirtangajjar) [reply src/Site_WP_Command.php]: Let me know if current limit isn't good enough

File: src/Site_WP_Command.php [hunk 2]
```diff
@@ -242,6 +242,17 @@ public function create( $args, $assoc_args ) {
...
 
+	/**
+	 * Creates database user for a site
+	 *
+	 * @param string $site_url URL of site
+	 *
+	 * @return string Generated db user
+	 */
+	private function create_site_db_user( string $site_url ) : string {
+		return substr( $site_url, 0, 73 ) . '-' . \EE\Utils\random_password( 6 );
```
Comment (rahulsprajapati) [inline src/Site_WP_Command.php]: do not directly do substr first please check if the string has that much char.  And let's not keep limit of 60.
```
if ( strlen( $site_url ) > 53 ) {
      $site_url = substr( $site_url, 0, 53 );
}
return $site_url . '-' . \EE\Utils\random_password( 6 );
```

Comment (kirtangajjar) [reply src/Site_WP_Command.php]: Done

**Pull request merged**